### ISSUE TITLE
Detect transaction with larger gas usage

### ISSUE THREAD

**Issue by ggrieco-tob** [crytic/medusa#163]
Once #75 is merged, we should generalize the code to allow medusa to show the most expensive transaction in terms of gas used.

**Comment by Xenomega** [comment]
Idea: The fuzzer_metrics.go file is pretty simple/sad. We could revamp it to track this and other statistics there. And leave room for arbitrary statistics users might have in their tests.

**Comment by anishnaik** [comment]
@ggrieco-tob putting this on hold for now. The PR lgtm but de-prioritizing this for a future milestone. Lmk if you disagree.

**Comment by anishnaik** [comment]
Removing this from "on hold" to low priority. We should try to close this out and review the PR in the next few milestones.

**Comment by anishnaik** [comment]
unlikely we implement
